connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -177,16 +180,14 @@ public class ErrorHandlingTaskTest {
 
     private ErrorHandlingMetrics errorHandlingMetrics;
 
-    private boolean enableTopicCreation;
+    private boolean enableTopicCreation = false;
 
+    private MockitoSession mockitoSession;
     @ParameterizedTest.Parameters
     public static Collection<Boolean> parameters() {
         return Arrays.asList(false, true);
     }
 
-    public ErrorHandlingTaskTest(boolean enableTopicCreation) {
```

Comment:
I had to remove this, since test class should have zero argument constructor (I got this error message while trying to run the test case class). 

I have set the enableTopicCreation value as false for now.